### PR TITLE
feat: add settings saved notice

### PR DESCRIPTION
**Pull request by j0hnfl0w**
Fixes https://github.com/snapshot-labs/snapshot/issues/3528

### Changes 
1. Added ModalNotice to SpaceSettings
2. Don't show again boolean saved to ls

### How to test
1. Go to Space setting and save some changes
2. Notice modal will be opened, if you click "Don't show again" it will not be shown again after space settings saved.

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [ ] I have tested my changes on a custom domain
- [ ] I have run end-to-end tests `yarn cypress:test:e2e`, and they have passed

### CODE REVIEW

**Comment by samuveth** [comment]
image

1. Utilize the [information message block](https://github.com/snapshot-labs/snapshot/blob/develop/src/components/BaseMessageBlock.vue) for conveying the message.
2. Position the checkbox at the lower section of the container.
3. Ensure that both the checkbox and message text are left-aligned.

File: src/locales/default.json [hunk 1]
```diff
@@ -293,6 +293,9 @@
     "confirmLeaveMessage": " Do you really want to leave? You have unsaved changes that will be lost.",
     "confirmDeleteSpace": " Do you really want to delete this space? This action cannot be undone.",
     "confirmInputDeleteSpace": "Enter {space} to continue:",
+    "noticeSettingsSavedTitle": "Settings saved",
```
Comment (samuveth) [inline src/locales/default.json:296]: We can remove settings from the naming as it is already nested inside the settings object/

File: src/views/SpaceSettings.vue [hunk 2]
```diff
@@ -361,5 +371,22 @@ const isViewOnly = computed(() => {
         </BaseInput>
       </div>
     </ModalConfirmAction>
+    <ModalNotice
```
Comment (samuveth) [inline src/views/SpaceSettings.vue:374]: Something weird happening here on small screen with the line break. Can you reproduce? 
image

Comment (j0hnfl0w) [reply src/views/SpaceSettings.vue:374]: Will remove br and v-html line breaks good without it

**Comment by samuveth** [comment]
👍 @j0hnfl0w

**Pull request merged**